.changeset/khaki-swans-add.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-birthday-picker": patch
```

Comment:
Did you meant to have two changeset files?

Reply:
ahh oops! I thought the `patch` needed to be handled separately. I'll remove this file.